include/ShaderWriter/Writer.hpp

```diff
@@ -630,6 +631,154 @@ namespace sdw
  ).endSwitch();\
 	}
 
+#define LocaleFloat( Writer, Name )\
```

Comment:
I had those macros in the first implementation, and I've intently removed them.
I'd rather not reintroduce them, sorry ^^'.